packages/cli/src/cli-commands/cli-version-commands.ts

```diff
@@ -281,8 +281,9 @@ export default {
         describe: 'Runs `npm install --package-lock-only` or equivalent depending on the package manager defined in `npmClient`.',
         type: 'boolean',
       },
-      'skip-bump-only-release': {
-        describe: 'do we want to skip creating a release (github/gitlab) when the version is a "version bump only"?',
+      'skip-bump-only-releases': {
+        describe: 'Skip release(s) creation when the version is a "version bump only"?',
+        alias: 'skip-bump-only-release',
```

Comment:
is this really needed? I don't think that the singular option will do anything because the option in VersionCommand is really plural not singular, so I'm assuming that this alias will do.. just nothing

https://github.com/lerna-lite/lerna-lite/blob/8d028847e8237a9b8ab94e36b23050aae219629c/packages/version/src/version-command.ts#L410-L413

Reply:
I removed the alias.
I think this is a breaking change. Users who previously used the singular argument will now receive an error message. But this may be even better than the current state (command executes, but argument has no effect).
In the Conventional Commit sense, however, this change should then lead to a major release, I think.